### PR TITLE
#5 Create project structure

### PR DESCRIPTION
**Pull request by ivshapovalov**
Connect #5

### CODE REVIEW

File: links-slackbot/src/main/java/juja/microservices/links/slackbot/Application.java [hunk 1]
```diff
@@ -1,15 +1,41 @@
...
- * @author Danil Kuznetsov (kuznetsov.danil.v@gmail.com)
- */
 @SpringBootApplication
 public class Application {
     public static void main(String[] args) {
         SpringApplication.run(Application.class, args);
     }
+
+    @Bean
+    public RestTemplate restTemplate() {
```
Comment (adaleorion) [inline links-slackbot/src/main/java/juja/microservices/links/slackbot/Application.java]: Why do we need resttemplate ?

Comment (ivshapovalov) [reply links-slackbot/src/main/java/juja/microservices/links/slackbot/Application.java]: I don't remember why somebody add all this methods in all Application main class. Maybe remove this and see what bad happens?

Comment (adaleorion) [reply links-slackbot/src/main/java/juja/microservices/links/slackbot/Application.java]: We used manual bean for add additional configuration We don't need this resttemplate  We use feign client

Spring boot creates this bean automatically with default settings

**Review by adaleorion — APPROVED** [review]
thanks, looks nice

**Pull request merged**